### PR TITLE
feat(EntityLayout): add 'if' prop to Route component

### PR DESCRIPTION
**Pull request by kuangp**
Signed-off-by: Phil Kuang <pkuang@factset.com>

## Hey, I just made a Pull Request!

Closes: #5296 
#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [x] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: plugins/catalog/package.json [hunk 1]
```diff
@@ -33,6 +33,7 @@
     "@backstage/catalog-client": "^0.3.10",
     "@backstage/catalog-model": "^0.7.7",
     "@backstage/core": "^0.7.6",
+    "@backstage/core-api": "^0.2.17",
```
Comment (Rugvip) [inline plugins/catalog/package.json:36]: `core-api` is internal and should not be used directly, import from `core` instead

**Review by Rugvip — COMMENTED** [review]
Awesome, looks good, thanks! 👍 😁 

Just a minor nit but then good to go

**Comment by freben** [comment]
i was keen on getting this into today's release, so added a fix on top - i hope that's ok

File: plugins/catalog/package.json (hunk 1 above)
Comment (freben) [reply plugins/catalog/package.json]: fixed

**Comment by backstage-service** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr5459-4lge0yn5ky0wkp9ct484z2fckdardhmp.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/608a91a78e7af66c39cbe3ac

**Review by benjdlambert — APPROVED** [review]

**Comment by kuangp** [comment]
@freben thanks for updating for me!

**Pull request merged**